src/back-end/AdoptApi/Services/UserService.cs

```diff
@@ -202,8 +203,16 @@ private static UserDto GetUserDto(User user)
             }
             user.Name = ChangeIfEmptyField(userEditDto.Name,user.Name);
             user.Email = ChangeIfEmptyField(userEditDto.Email,user.Email);
+            if (user.Document.Type == DocumentType.Cnpj)
+            {
+                user.Address.Name = ChangeIfEmptyField(userEditDto.Address,user.Address.Name);
+                user.Address.Number = Int32.Parse(ChangeIfEmptyField(userEditDto.Number.ToString(),user.Address.Name.ToString()));
```

Comment:
Possível bug: nesta linha, se um número não for informado, o default será o NOME do endereço.

Reply:
Eita kkkk